### PR TITLE
added reading list

### PR DESCRIPTION
**Pull request by Lewis-Seiden**

### CODE REVIEW

File: ReadingList.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Reading List
+
+This is a collection of books and articles to continue learning beyond this repo.
+Most of these are suggestions by Kevin right now, but if you have something feel free to add it.
+
+- Head First Design Patterns
```
Comment (Awesomeplayer165) [inline ReadingList.md:6]: - [Head First Design Patterns](https://www.amazon.com/Head-First-Design-Patterns-Brain-Friendly/dp/0596007124)

File: ReadingList.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+- The Pragmatic Programmer - Thomas & Hunt
```
Comment (Awesomeplayer165) [inline ReadingList.md:7]: - [The Pragmatic Programmer - Thomas & Hunt](https://www.amazon.com/Pragmatic-Programmer-Journeyman-Master/dp/020161622X)

File: ReadingList.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+- Refactoring - Martin Fowler
```
Comment (Awesomeplayer165) [inline ReadingList.md:8]: - [Refactoring - Martin Fowler](https://www.amazon.com/Refactoring-Improving-Design-Existing-Code/dp/0201485672)

File: ReadingList.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+- Effective Java - Josh Bloch
```
Comment (Awesomeplayer165) [inline ReadingList.md:9]: - [Effective Java - Josh Bloch](https://www.amazon.com/Effective-Java-Joshua-Bloch/dp/0134685997)

**Review by Awesomeplayer165 — CHANGES_REQUESTED** [review]
Consider adding links to the actual references

**Comment by Lewis-Seiden** [comment]
i can find some pdf links

**Pull request merged**